From fd7a8850604116d57e3802e5ea3e8c5f08623a85 Mon Sep 17 00:00:00 2001
From: Grace Si <b18730@freescale.com>
Date: Thu, 10 May 2012 17:06:21 +0800
Subject: [PATCH 598/670] ENGR00182404-2 [MX28] kernel 5V only configuration
 bug fixes and improvement

Change VDDA voltage from 1.75V to 1.8V which is the typical figure.

Signed-off-by: Grace Si <b18730@freescale.com>
---
 arch/arm/mach-mx28/bus_freq.c | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm/mach-mx28/bus_freq.c b/arch/arm/mach-mx28/bus_freq.c
index 920c630..7687242 100644
--- a/arch/arm/mach-mx28/bus_freq.c
+++ b/arch/arm/mach-mx28/bus_freq.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2010 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2012 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -51,26 +51,26 @@
 #ifdef CONFIG_MEM_mDDR
 struct profile profiles[] = {
 	{ 454736, 151570, 205710, 0, 1550000,
-	1450000, 355000, 3300000, 1750000, 24000, 0 },
+	1450000, 355000, 3300000, 1800000, 24000, 0 },
 	{ 360000, 120000, 130910, 0, 1350000,
-	1250000, 200000, 3300000, 1750000, 24000, 0 },
+	1250000, 200000, 3300000, 1800000, 24000, 0 },
 	{ 261818, 130910, 130910, 0, 1350000,
-	1250000, 173000, 3300000, 1750000, 24000, 0 },
+	1250000, 173000, 3300000, 1800000, 24000, 0 },
 	{  64000,  64000, 240000, 3, 1350000,
-	1250000, 150000, 3300000, 1750000, 24000, 0 },
+	1250000, 150000, 3300000, 1800000, 24000, 0 },
 	{  0,  0,  0, 0, 0,
 	0, 0, 0, 0, 0, 0 },
 };
 #else
 struct profile profiles[] = {
 	{ 454736, 151570, 205710, 0, 1550000,
-	1450000, 355000, 3300000, 1750000, 24000, 0 },
+	1450000, 355000, 3300000, 1800000, 24000, 0 },
 	{ 360000, 120000, 130910, 0, 1350000,
-	1250000, 200000, 3300000, 1750000, 24000, 0 },
+	1250000, 200000, 3300000, 1800000, 24000, 0 },
 	{ 261818, 130910, 130910, 0, 1350000,
-	1250000, 173000, 3300000, 1750000, 24000, 0 },
+	1250000, 173000, 3300000, 1800000, 24000, 0 },
 	{  64000,  64000, 130910, 3, 1350000,
-	1250000, 150000, 3300000, 1750000, 24000, 0 },
+	1250000, 150000, 3300000, 1800000, 24000, 0 },
 	{  0,  0,  0, 0, 0,
 	0, 0, 0, 0, 0, 0 },
 };
-- 
1.8.0

